.flaskenv

```diff
@@ -1,5 +1,5 @@
 FLASK_APP=time_tracker.py
-FLASK_ENV=development
```

Comment:
Deprecated in `2.2.0`: https://flask.palletsprojects.com/en/2.2.x/changes/#version-2-2-0